packages/snowpack/src/build/build-pipeline.ts

```diff
@@ -67,7 +69,9 @@ async function runPipelineLoadStep(
         isHmrEnabled,
         // @ts-ignore: internal API only
         log: (msg, data: any = {}) => {
-          if (data && data.msg) pluginLogger.info(`${data.msg} [${debugPath}]`);
+          if (data && data.msg) {
```

Comment:
this is the only hook where we actually still need the message bus (`run()` is the only hook allowed to own it's output area, ex: typescript).

the Svelte CSA template does work, but the Svelte TypeScript template doesn't, which I think may be coming from here.